### PR TITLE
Cv 303 edit draft apprenticeship page

### PR DESCRIPTION
**Pull request by Pauljgraham**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -14,10 +14,10 @@ variables:
   buildPlatform: 'anycpu'
 
 pool:
-  name: 'Continuous Integration 02 - SSD - 160ACU'
+  name: 'DAS - Continuous Integration'
```
Comment (chrisfoster76) [inline azure-pipelines.yml]: Are these changes deliberate?

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/DraftApprenticeship/EditDraftApprenticeship.cshtml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+@{
+    ViewData["Title"] = "Apprenticeships";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ValidationSummary" />
+        <h1 class="govuk-heading-xl">Edit apprentice details</h1>
+        <div class="govuk-inset-text">
```
Comment (chrisfoster76) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Views/DraftApprenticeship/EditDraftApprenticeship.cshtml]: This inset text is incorrect - but there is no screenshot for us to build against in the ticket

File: azure-pipelines.yml (hunk 1 above)
Comment (Pauljgraham) [reply azure-pipelines.yml]: these were taken from another branch

**Comment by Pauljgraham** [comment]
merged in as part of PR CV-247
